cmd/build.go

```diff
@@ -96,6 +129,32 @@ func doBuild(opts buildOpts) (*dag.DAG, error) {
 		},
 	}
 
+	if opts.backend == backendKaniko {
+		var executor kaniko.Executor
```

Comment:
Maybe we could extract the kanikoBuilder setup in its own function ? from line 134 to 153